ethereum/src/Relayer.sol

```diff
@@ -105,4 +122,24 @@ contract Relayer is IRelayer {
 
         return ret;
     }
+
+    /**
+     * @dev Checks if the merkle root provided is duplicated or not.
+     * If it is duplicated, it sets the emergency mode.
+     *
+     * @param block_number Target block number.
+     * @param merkle_root Target merkle root.
+     */
+    function _checkEmergencyStopCondition(
```

Comment:
I think it can be inlined and done without `if` condition

Reply:
Are you suggest to inline this function?

Reply:
yes